bob-common/src/mapper.rs

```diff
@@ -121,6 +121,44 @@ impl Virtual {
         self.vdisks.get(&id).expect("vdisk not found").nodes()
     }
 
+    fn find_index(&self, offset: usize) -> Offset {
```

Comment:
Expand the name to make it clear that the function is for the search for the support node

Reply:
It is better to make function static: don't receive `self` as parameter